index.html

```diff
@@ -2458,14 +2458,24 @@ <h3>Data Schemas</h3>
 
         <dl>
           <dt><var>credentialSchema</var></dt>
-          <dd>
-The value of the <code>credentialSchema</code> <a>property</a> MUST be one or
-more data schemas that provide <a>verifiers</a> with enough information to
-determine if the provided data conforms to the provided schema. Each
-<code>credentialSchema</code> MUST specify its <code>type</code> (for example,
-<code>JsonSchemaValidator2018</code>), and an <code>id</code> <a>property</a>
-that MUST be a <a>URL</a> identifying the schema file. The precise contents of
-each data schema is determined by the specific type definition.
+          <dd><p>
+The value of the <code>credentialSchema</code> <a>property</a> MUST be a single
+data schema object which provides <a>verifiers</a> with enough information 
```

Comment:
Hmm, this makes it not possible to provide more than one type of schema... for example, one schema to do data validation (e.g., JSON Schema) and another schema to do data transformation (e.g., CL Signatures, Gordian Envelopes, etc.).

Limiting to ONE and only one schema is probably not a good idea. 

Limiting to one schema type in the list of `credentialSchema`s is probably better.

Reply:
> Limiting to ONE and only one schema is probably not a good idea.

If we don't limit to one, we must provide examples that make use of more than one, and tests...

I suggest limiting to one.

Reply:
Yes - suggest limiting to one schema type in the list of credential schema, which when resolved can contain multiple schemas